platform/linux-generic/include/odp_packet_internal.h

```diff
@@ -149,6 +149,10 @@ typedef struct ODP_ALIGNED_CACHE odp_packet_hdr_t {
 	/* LSO profile index */
 	uint8_t lso_profile_idx;
 
+	/* Packet aging drop timeout before enqueue. Once enqueued holds the maximum age (time of
+	 * request + requested drop timeout). */
+	uint64_t tx_aging_ns;
```

Comment:
I'd swap this and `lso_profile_idx`, so the gap in struct would be in a clearer location.

Reply:
OK, swapped in v3. Didn't do any calculations, hopefully alignment is better now.